src/Game.java

```diff
@@ -0,0 +1,316 @@
+        System.out.println("You can move the pig by pressing arrows. ");
+        System.out.println("Different keys plant different crops.\nQ: Potatoes\nW: Apples\nE: Oranges\nR: Cabbages");
+        System.out.println("If you gained enough money, you can choose to unlock 1 of 3 plants every 5 days.");
+        System.out.println("If you gained enough money, you can choose to unlock 1 of the lands every 10 days.");
+        System.out.println("You can press the 'skip day' to skip day and make the plants grow faster.");
+        System.out.println("You can press the space bar to harvest the plants and gain money.");
+        System.out.println("However, you can only plant 9 items in one land. ");
+        System.out.println("Enjoy this game!😂");
+        new Game();
+    }
+
+    /** This creates a rectangle which informs the user of what day it is.
+    */
+    public void timeButton() {
+        timeButton = new Rectangle(0, 0, 95, 40);
+        timeButton.setCenter(815, 150);
+        timeButton.setFillColor(Color.RED);
+        timeLabel.setText("Day: " + days);
+        timeLabel.setFont(FontStyle.BOLD, 15);
+        timeLabel.setFillColor(Color.WHITE);
+        timeLabel.setCenter(815, 150);
+    }
+
+    /**
+     * This creates a clickable button called 'Skip Day' which allows users to move on to the next day in the game.
+     * 
+     */
+    public void Button() {
+        button = new Button("Skip Day");
+        button.setCenter(815, 200);
+        canvas.add(button);
+        button.onClick(() -> skipDayMechanism());
+    }
+
+    /**This method grows plants and asks several different questions depending on what day it is in the game. */
+    public void skipDayMechanism() {
+        for (Plant plant : character.getList()) {
+            if (!plant.maxSize()) {
+                plant.growLarger();
+            }
+        }
+        askAboutLand();
+        askAboutPlant();
+        days++;
+        changeDay();
+    }
+
+    /**
+     * This method asks the player if they'd like to unlock a new piece of land.
+     * The question only pops up if they have the money to buy land,
+     * and it will stop popping up entirely when there is no more land to unlock (check lockCounter)
+     * */
+    public void askAboutLand() {
+        if (days % 10 == 0) {
+            scanner = new Scanner(System.in);
+
+            if (character.getMoney() >= 100 && lockCounter <= 6) {
+                System.out.println("Do you wanna unlock land(Yes/No)");
+                String unlockResponse = scanner.nextLine();
+                if (unlockResponse.equalsIgnoreCase("Yes")) {
+                    environment.getLands().get(lockCounter + 3).unlock();
+                    lockCounter++;
+                    character.subtractMoney(100);
+                    changeMoney();
+                } else if (unlockResponse.equalsIgnoreCase("No")) {
+                    return;
+
+                }
```

Comment:
It would really help the playability of the game to put this in a loop that retries until it gets a valid answer, so that a type or extra input doesn’t rob a user of their chance to answer.